Vagrantfile

```diff
@@ -113,6 +129,9 @@ Vagrant.configure("2") do |config|
 		]
 	end
 
+	# Show the URL and login details.
+	config.vm.post_up_message = Chassis.post_up_message
```

Comment:
I might revert this change as I ended up with duplicated outputs because of original script.